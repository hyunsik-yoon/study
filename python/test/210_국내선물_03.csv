거래일자,거래번호,거래종류,적요명,종목번호,종목명,수량,단가,거래금액,수수료,예수금전잔,예수금금잔,정산금액,약정금액,세금합계,대용잔고,기타수표금액,미수/변제,대체금융회사,계좌명,받는분 표시내용,보내는분 표시내용
2023-01-12,3,매수,당일차금매수입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000","21,850","500,000","21,130","44,169,017","44,647,887","478,870","218,500,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-13,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"1,500,000",0,"44,647,887","43,147,887","1,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-16,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"2,500,000",0,"43,147,887","40,647,887","2,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-17,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"1,500,000",0,"40,647,887","39,147,887","1,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-18,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"3,500,000",0,"39,147,887","35,647,887","3,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-19,1,입금,매수갱신차금입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"2,000,000",0,"35,647,887","37,647,887","2,000,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-20,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"3,000,000",0,"37,647,887","34,647,887","3,000,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-25,1,입금,매수갱신차금입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"2,500,000",0,"34,647,887","37,147,887","2,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-26,1,입금,매수갱신차금입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"2,500,000",0,"37,147,887","39,647,887","2,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-27,1,입금,매수갱신차금입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"1,500,000",0,"39,647,887","41,147,887","1,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-30,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"3,000,000",0,"41,147,887","38,147,887","3,000,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-01-31,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"6,000,000",0,"38,147,887","32,147,887","6,000,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-01,1,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,000",0,"1,500,000",0,"51,548,595","50,048,595","1,500,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-01,2,매수,당일차금매수입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,20,"20,450","10,000",0,"50,048,595","50,058,595","10,000","4,090,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-01,3,매수,당일차금매수출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,40,"20,500",0,"1,180","50,058,595","50,057,415","1,180","8,200,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-02,1,입금,매수갱신차금입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,060",0,"3,710,000",0,"50,057,415","53,767,415","3,710,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-03,1,입금,매수갱신차금입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,060",0,"2,120,000",0,"53,767,415","55,887,415","2,120,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-06,1,입금,매수갱신차금입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,"1,060",0,"2,650,000",0,"57,138,100","59,788,100","2,650,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-06,3,매도,당일차금매도입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,120,"21,400","120,000","2,480","62,978,960","63,096,480","117,520","25,680,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-07,3,출금,매수갱신차금출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,940,0,"940,000",0,"57,016,980","56,076,980","940,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-07,9,매도,당일차금매도입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,940,"21,250","470,000","19,310","57,687,970","58,138,660","450,690","199,750,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-09,3,매도,당일차금매도출금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,100,"23,350","400,000","2,250","88,120,230","87,717,980","402,250","23,350,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
2023-02-09,4,매수,당일차금매수입금                                                                ,1foo2000,foobar   F 202302 (  10)                                                                                                                                                                                                                                                                                  ,100,"23,400","350,000","2,260","87,717,980","88,065,720","347,740","23,400,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,,
