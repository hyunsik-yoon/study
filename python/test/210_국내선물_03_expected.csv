종목명,진입일자,진입단가,청산일자,청산단가,거래수량,총수수료,총세금합계,순수익
foobar   F 202302 (  10),01/12/2023,"21,850",02/06/2023,"21,400",120,"5,016",0,-540000
foobar   F 202302 (  10),01/12/2023,"21,850",02/07/2023,"21,250",880,36672,0,-5280000
foobar   F 202302 (  10),02/01/2023,"20,450",02/07/2023,"21,250",20,411,0,160000
foobar   F 202302 (  10),02/01/2023,"20,500",02/07/2023,"21,250",40,2002,0,300000
foobar   F 202302 (  10),02/09/2023,"23,350",02/09/2023,"23,400",100,4510,0,-50000
